pycheribuild/config/target_info.py

```diff
@@ -729,6 +784,10 @@ class CompilationTargets(object):
     FREEBSD_AARCH64 = CrossCompileTarget("aarch64", CPUArchitecture.AARCH64, FreeBSDTargetInfo)
     FREEBSD_X86_64 = CrossCompileTarget("x86_64", CPUArchitecture.X86_64, FreeBSDTargetInfo)
 
+    # RTEMS targets
+    RTEMS_NEWLIB_RISCV64 = CrossCompileTarget("rtems-riscv64", CPUArchitecture.RISCV64,
```

Comment:
I would probably call this rtems-riscv64-purecap so we can add a hybrid/plain riscv version later.
```suggestion
    RTEMS_NEWLIB_RISCV64 = CrossCompileTarget("rtems-riscv64-purecap", CPUArchitecture.RISCV64,
```

Reply:
I don't think we will need to build newlib in vanilla and/or hybrid mode for RTEMS-CHERI-RISC-V in cheribuild, that's why I omitted it and assumed that target is always gonna be purecap